### PR TITLE
[CodeQuality] Skip SimplifyDeMorganBinaryRector on negated greater or equal

### LINKED ISSUE #5153: Incorrect behavior of SimplifyDeMorganBinaryRector
# Bug Report

| Subject        | Details              |
| :------------- | :--------------------|
| Rector version | last dev-master      |
| Installed as   | composer dependency  |

Negated conditions was changed from `!($c >= $a + $b)` to `!$c < !($a + $b)`.

## Minimal PHP Code Causing Issue

See https://getrector.org/demo/68a3508e-c718-4f99-86ed-884d617e7e69

```php
<?php

function demo(int $a, int $b, int $c): bool
{
    if (! ($c >= $a + $b)) {
        return true;
    }
    return false;
}
```

### Responsible rules

* `SimplifyDeMorganBinaryRector`

## Expected Behavior

Nothing needs to be changed

### PR DESCRIPTION
**Pull request by samsonasik**
Fixes #5153

### CODE REVIEW

**Comment by samsonasik** [comment]
All checks have passed 🎉 @TomasVotruba it is ready for review.

File: rules/code-quality/tests/Rector/BooleanNot/SimplifyDeMorganBinaryRector/Fixture/skip_greater_or_equal.php.inc [hunk 1]
```diff
@@ -0,0 +1,15 @@
+<?php
+
```
Comment (TomasVotruba) [inline rules/code-quality/tests/Rector/BooleanNot/SimplifyDeMorganBinaryRector/Fixture/skip_greater_or_equal.php.inc:2]: Here should be namespace of the fixture as well.

Comment (samsonasik) [reply rules/code-quality/tests/Rector/BooleanNot/SimplifyDeMorganBinaryRector/Fixture/skip_greater_or_equal.php.inc:2]: fixed.

**Comment by TomasVotruba** [comment]
Thanks :+1:

**Pull request merged**